migrations/2018_01_19_084700_change_settings_value_longblob.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $prefix = $schema->getConnection()->getTablePrefix();
+//        $schema->getConnection()->statement('ALTER TABLE '.$prefix.'settings MODIFY "value" LONGBLOB');
+    },
+
+    'down' => function (Builder $schema) {
+        $schema->table('posts', function (Blueprint $table) {
+//            $table->longText('value')->nullable()->change();
```

Comment:
?

Reply:
Fixed in https://github.com/flarum/core/pull/1344/commits/8779e40ec5ddf978b0bca25b51f470c1a04153f5,

we decided to not change the column type (as discussed in the issue).